### ISSUE TITLE
BUG: Using scrutiny with a base path does not work

### ISSUE THREAD

**Issue by tandy-1000** [AnalogJ/scrutiny#257]
I am using caddy as a reverse proxy.

Part of my scrutiny config:
```
  listen:
    host: 0.0.0.0
    port: 9293
    basepath: '/scrutiny'
  src:
    frontend:
      path: /opt/scrutiny/web
```

Firefox console log:
```
09:47:14.771 Loading module from “http://server.local/web/polyfills-es2015.5483988857642b9a2626.js” was blocked because of a disallowed MIME type (“text/html”).
[web](http://server.local/scrutiny/web/)
09:47:14.772 Loading failed for the module with source “http://server.local/web/polyfills-es2015.5483988857642b9a2626.js”. [web:146:1](http://server.local/scrutiny/web/)
09:47:14.772 Loading module from “http://server.local/web/main-es2015.0b3088a2500f925ce084.js” was blocked because of a disallowed MIME type (“text/html”).
[web](http://server.local/scrutiny/web/)
09:47:14.772 Loading failed for the module with source “http://server.local/web/main-es2015.0b3088a2500f925ce084.js”. [web:146:1](http://server.local/scrutiny/web/)
09:47:14.772 Loading module from “http://server.local/web/runtime-es2015.70229d90c812da71ec53.js” was blocked because of a disallowed MIME type (“text/html”).
[web](http://server.local/scrutiny/web/)
09:47:14.772 Loading failed for the module with source “http://server.local/web/runtime-es2015.70229d90c812da71ec53.js”.
```

**Comment by AnalogJ** [comment]
Are you sure this isnt due to a mis-configuration in Caddy? 
I just spun up the latest omnibus image and I cant seem to replicate that issue (with firefox or chrome)

```
docker run -it --rm -p 8080:8080 -p 8086:8086 \
  -v /run/udev:/run/udev:ro \
  --cap-add SYS_RAWIO \
  --name scrutiny \
  -e SCRUTINY_WEB_LISTEN_BASEPATH=/testing \
  ghcr.io/analogj/scrutiny:master-omnibus
```

I didnt see anything like that in the console, even with a clean cache. 

Are you running the web process on Windows? What is the output of `go env` on your machine?

**Comment by AnalogJ** [comment]
Actually, I'm a bit confused. I see that you set a basepath. Shouldn't your requested url be `http://server.local/scrutiny/web/`?

**Comment by tandy-1000** [comment]
I am visiting `http://server.local/scrutiny/web/`
I am running Scrutiny on DietPi (Debian).
My caddy config looks something like this:
```
:80 {
  encode zstd gzip
  ...
  reverse_proxy /scrutiny* :9293
  ...
}
```

**Comment by AnalogJ** [comment]
@tandy-1000 would you mind sharing over a quick docker-compose file that would allow me to replicate this issue locally?

I don't use Caddy so its a bit hard for me to debug.

**Comment by AnalogJ** [comment]
fixed in v0.4.8 🎉
